### DISCUSSION TITLE
Второй пояс для патронов, и перенос пояса для артов из ЗП, с количеством ячеек в зависимости от костюмов.

### DISCUSSION THREAD

**Discussion by D1m0n93** [OGSR/OGSR-Engine#293] [Ideas]
Оставлю это тут, вдруг когда-нить запилят...

**Comment by xrSimpAI** [comment]
_Для информации: На данный момент эта система не рассчитана на арты, занимающие больше одной ячейки. Так что имейте ввиду, что всякие колбы по три ячейки, патроны и тп. на пояс сувать не надо, результат будет печальным._
TODO: что-то придумать с этим, или написать в wiki, чтоб так не делали.

**Comment by joye-ramone** [comment]
это все жутко усложнит. проще разрешать вещать на пояс всегда айтемы по одной ячейке

**Comment by Kirito03** [comment]
На примере Нового Арсенала 7: там многие иконки патронов (кроме снайперских, пулемётных и ружеёных) заменили на одноячеичные для совместимости - такой вот _костыль_ к ОГСР-движку
